### ISSUE TITLE
"bad request" when trying to set a skin via an url

### ISSUE THREAD

**Issue by melzyy** [Suiranoil/SkinRestorer#28]
Any valid link doesn't work and the mod prints "bad request" in chat with no details. 

Another issue is skins not setting via the mojang option, but thats because my IP is blocked by mojang servers for some reason, trying to sort that out too, not the mod's problem probably

**Comment by Suiranoil** [comment]
It is difficult to tell whats wrong without an example.
Make sure your link is a direct link to your image, for example, this is how it looks like for imgur: https://i.imgur.com/3rDMtY2.png. You can usually get it by right clicking your image in browser and selecting something like "Copy image address".

**Comment by melzyy** [comment]
I've requested to change the IP of my server to the IP that is not banned my Mojang and the issues solved themselves. Strange that it also affected the alternative image hosts lol. Closing the issue as it is fixed.
